Gek62/hometask-4/my_hash.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Hash class extension created
+class MyHash
```

Comment:
Please, rewrite and make sure that your hash works exactly the same way as `Hash`, because at the moment it does not.